docs/docs-content/clusters/public-cloud/azure/required-permissions.md

```diff
@@ -309,659 +263,1326 @@ various scenarios.
 - The Azure Security Principal Object ID you want to use with Palette. The Security Principal Object ID can represent a
   user, group, or service principal.
 
-### Create Role and Assign Permissions
+#### Create Role and Assign Permissions
+
+1. Open a terminal or command prompt. Export your Azure subscription ID to a variable.
+
+   ```shell
+   export SUBSCRIPTION_ID=<your-subscription-id>
+   ```
+
+2. Issue the following command to create a JSON file containing all the required permissions to create resources.
+
+   ```shell
+   cat << EOF > iaas_dynamic_rg_sub_role.json
+   {
+     "Name": "Palette Dynamic Placement IaaS Cluster Deployer (rg/sub)",
+     "IsCustom": true,
+     "Description": "Can use Palette to deploy IaaS clusters using static placement. This role contains the permissions that must be applied at the resource group scope level. If deploying multiple clusters in a variety of resource groups within a subscription, apply the role with the subscription as scope instead of the resource group as scope.",
```

Comment:
Looks like it the title says "dynamic" but the desc says "static". This might be from the content I handed over though. Should double check the rest of this file for that sort of thing.